src/components/user/row.vue

```diff
@@ -52,8 +52,9 @@ except according to the terms contained in the LICENSE file.
               {{ $t('action.resetPassword') }}&hellip;
             </a>
           </li>
-          <li :class="{ disabled }" :title="disabled ? $t('cannotRetire') : null">
+          <li :class="{ disabled }">
             <a class="retire-user" href="#"
+              v-tooltip.aria-describedby="disabled ? $t('cannotRetire') : null"
               @click.prevent="$emit('retire', user)">
               {{ $t('action.retire') }}&hellip;
             </a>
```

Comment:
Similar to the tabs in `FormHead`, the `<li>` had the `title` attribute before, but now I've put the tooltip on the link. I don't think it makes a big difference for users using a mouse, but for screen readers, I think it's helpful for the link to get the `aria-describedby` attribute.